modules/va_notify/app/services/va_notify/email_delivery_status_callback.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module VANotify
```

Comment:
Nice of you to create `VANotify::EmailDeliveryStatusCallback` for everyone!

If the VANotify team is cool with this they should update their docs by pointing to this.